include/xtensor-r/rcontainer.hpp

```diff
@@ -110,12 +142,12 @@ namespace xt
         using difference_type = typename storage_type::difference_type;
 
 #ifndef XTENSOR_R_ALLOW_REINTERPRETATION
-        static_assert(xtl::disjunction<std::is_same<value_type, int32_t>,
-                                       std::is_same<value_type, double>,
-                                       std::is_same<value_type, Rbyte>,
-                                       // std::is_same<value_type, bool>, NOT YET IMPLEMENTED!
-                                       std::is_same<value_type, std::complex<double>>>::value == true,
-                      "R containers can only be of type bool, int, double, complex<double>.");
+        static_assert(xtl::disjunction<std::is_same<r_type, int32_t>,
+                                       std::is_same<r_type, double>,
+                                       std::is_same<r_type, Rbyte>,
+                                       std::is_same<r_type, rlogical>,
+                                       std::is_same<r_type, std::complex<double>>>::value == true,
+                      "R containers can only be of type logical, int, double, complex<double>.");
```

Comment:
maybe write `rlogical` in the error message? Actually, we also should write `std::complex<double>`

Reply:
Fixed both of these